TestVectorsAwsCryptographicMaterialProviders/Makefile

```diff
@@ -153,6 +172,19 @@ TRANSLATION_RECORD_PYTHON := \
     --translation-record ../AwsCryptographyPrimitives/runtimes/python/src/aws_cryptography_primitives/internaldafny/generated/dafny_src-py.dtr \
 	--translation-record ../AwsCryptographicMaterialProviders/runtimes/python/src/aws_cryptographic_material_providers/internaldafny/generated/dafny_src-py.dtr
 
+# Smithy-dafny generated shim needs a long term fix. 
+# Smithy-dafny generates polymorph code of MPL in test vectors.  
+# TODO: Remove these commands once smithy-dafny is fixed
+polymorph_go:
+	find .. -name "shim.go" | xargs sed -i "" 's/(_static \*CompanionStruct_Default___)//g'
```

Comment:
Does this work on all platforms?

Reply:
No, this does not work on windows. Test vectors already use commands that does not work on windows (for example: `cp`).